dashboard/app/views/levels/_single_multi.html.haml

```diff
@@ -67,20 +85,23 @@
   = render partial: 'levels/teacher_markdown', locals: {data: level.properties}
 
   %br/
-  %br/
-  %br/
+  - if standalone
+    %br/
+    %br/
 
   .clear
 
-:javascript
-
-  $(document).ready(function() {
-    window["level_#{index}"] = new Multi(
-      "level_#{index}",
-      #{level.id},
-      #{standalone},
-      "#{data['type']}" == "multi2" ? 2 : 1,
-      #{data['answers'].map {|answer| answer['correct']}},
-      #{last_attempt.to_json}
-      );
-    });
+-# Don't operate if we're a LevelGroup with a /level/ URL.
+- unless !standalone && @script_level.nil?
```

Comment:
too many negatives for me to understand :) change to `if standalone || @script_level`? (is that right?)